### ISSUE TITLE
Plugin has issues with Vite 3.1.X, parse errors

### ISSUE THREAD

**Issue by JoshuaHintze** [shixuanhong/vite-plugin-css-export#1]
I'm using 1.0.3 of the vite-plugin-css-export, this works with all my scss files just great with Vite 3.0.X. However when upgrading to Vite 3.1.X I now get scss parse errors as shown below.

```
 ERROR  3:23:52 PM [vite] Internal server error: Unexpected token (1:8)                                                                                                                                 15:23:52
  Plugin: vite:css-post
  File: XXX/test/src/themes/themes.scss?export
      at Parser.pp$4.raise (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:30629:13)
      at Parser.pp$9.unexpected (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:27930:8)
      at Parser.pp$9.expect (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:27924:26)
      at Parser.pp$5.parseExprList (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:30496:12)
      at Parser.pp$5.parseExprAtom (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:29984:26)
      at Parser.pp$5.parseExprSubscripts (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:29805:19)
      at Parser.pp$5.parseMaybeUnary (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:29771:17)
      at Parser.pp$5.parseExprOps (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:29698:19)
      at Parser.pp$5.parseMaybeConditional (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:29681:19)
      at Parser.pp$5.parseMaybeAssign (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:29648:19)
      at Parser.pp$5.parseExpression (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:29611:19)
      at Parser.pp$8.parseStatement (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:28120:45)
      at Parser.pp$8.parseTopLevel (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:27987:21)
      at Parser.parse (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:27759:15)
      at Function.parse (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:27809:35)
      at TransformContext.parse (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:40715:27)
      at TransformContext.vitePostCodeHandler (XXX/node_modules/vite-plugin-css-export/dist/index.cjs:127:20)
      at TransformContext.cssPostPlugin.transform (XXX/node_modules/vite-plugin-css-export/dist/index.cjs:160:38)
      at async Object.transform (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:41031:30)
      at async loadAndTransform (file://XXX/node_modules/vite/dist/node/chunks/dep-665b0112.js:37292:29)
```

Here is my vite.config

```
import { defineConfig } from 'vite';
import react from '@vitejs/plugin-react';
import { createHtmlPlugin } from 'vite-plugin-html';
import ViteCSSExportPlugin from 'vite-plugin-css-export';
import { imagetools } from 'vite-imagetools';

// https://vitejs.dev/config/
export default defineConfig({
	plugins: [
		react(),
		createHtmlPlugin({
			minify: true,
			entry: '/src/index.tsx',
			template: 'public/index.html'
		}),
		ViteCSSExportPlugin(),
		imagetools()
	],
	css: {
		// Adds Source Maps when viewing SCSS in dev mode
		devSourcemap: true
	},
	build: {
		// If enabled - the dist folder will contain map files, only good for sandbox
		// sourcemap: 'inline'
	},
	server: {
		// Uncomment when using ngrok. See this post: https://github.com/vitejs/vite/discussions/5399
		// hmr: { clientPort: 443 },
		port: 3000,
		proxy: {
		}
	}
});
```

Any suggestions? Thank you for the great plugin!

**Comment by JoshuaHintze** [comment]
Any thoughts on this? Are you not having issues?

**Comment by JoshuaHintze** [comment]
@shixuanhong

**Comment by shixuanhong** [comment]
> Any thoughts on this? Are you not having issues?

Fixed in version 1.1.0

**Comment by JoshuaHintze** [comment]
confirmed working. Thanks!
